101-Telegraf-InfluxDB-Grafana/azuredeploy.json

```diff
@@ -81,11 +82,77 @@
         }
       }
     },
+    {
+      "comments": "Default Network Security Group for template",
```

Comment:
need a way to access the vm

Reply:
Yes, I found an issue with the scan.  I'll update to add remote access.

Reply:
Updated.